app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -47,6 +49,18 @@ export default class AgendaitemDecisionComponent extends Component {
     this.previousReport = yield this.report?.previousPiece;
   }
 
+  @action
+  async onSaveDecisionResultCode() {
```

Comment:
The component hook is called `on...` but the action implementing the hook should not start with `on...` but reflect what will happen (eg. `updateDocumentAccessLevels`)